### PR TITLE
Adds heek table quit message

### PR DESCRIPTION
**Pull request by Hazado**
Script changes needed - https://github.com/H-uru/Plasma/pull/1446

Translation done with google translate.

### CODE REVIEW

**Review by dgelessus — APPROVED** [review]
German and French translations look okay to me.

File: compiled/dat/HeekEnglish.loc [hunk 1]
```diff
@@ -20,6 +20,9 @@
 			<element name="WelcomePlural">
 				<translation language="English">Please welcome %1s, a rank %2s player with %3s points!</translation>
 			</element>
+			<element name="Quit">
+				<translation language="English">Do you want to leave the Ahyoheek game?</translation>
```
Comment (Hoikas) [inline compiled/dat/HeekEnglish.loc:24]:
```suggestion
				<translation language="English">Do you want to forfeit leave Ahyoheek and forfeit this game?</translation>
```

Without bike shedding the message too much, I think it's important to indicate that the current round will be forfeited (meaning, you lose a point) if you quit.

Comment (Hazado) [reply compiled/dat/HeekEnglish.loc:24]: Currently its not coded to know if your in a game or not. You get this message as soon as you stand up. This message wont always be accurate

Comment (Hoikas) [reply compiled/dat/HeekEnglish.loc:24]: Sounds like something to improve in the code ;)

File: compiled/dat/HeekGerman.loc [hunk 2]
```diff
@@ -21,7 +21,7 @@
 				<translation language="German">Bitte heiße %1s willkommen, ein Spieler mit Rang %2s und %3s Punkten!</translation>
 			</element>
 			<element name="Quit">
-				<translation language="German">Möchtest du das Ahyoheek-Spiel verlassen?</translation>
+				<translation language="German">Möchten Sie das Ahyoheek-Spiel verlassen und diese Runde aufgeben?</translation>
```
Comment (dgelessus) [inline compiled/dat/HeekGerman.loc:24]:
```suggestion
				<translation language="German">Möchtest du das Ahyoheek-Spiel verlassen und diese Runde aufgeben?</translation>
```

**Pull request merged**